views/js/test/qtiCommonRenderer/interactions/selectPoint/qti.json

```diff
@@ -0,0 +1,182 @@
+{
+    "identifier": "i14862478187486450",
+    "serial": "item_589665700e6aa547084007",
+    "qtiClass": "assessmentItem",
+    "attributes": {
+        "identifier": "i14862478187486450",
+        "title": "Select Point Interaction",
+        "label": "Select Point Interaction",
+        "xml:lang": "en-US",
+        "adaptive": false,
+        "timeDependent": false,
+        "toolName": "TAO",
+        "toolVersion": "3.2.0-sprint43",
+        "class": ""
+    },
+    "body": {
+        "serial": "container_containeritembody_589665700e692347974427",
+        "body": "\n    <div class=\"grid-row\">\n      <div class=\"col-12\">\n        {{interaction_selectpointinteraction_5896657031abe915479781}}\n      <\/div>\n    <\/div>\n  ",
+        "elements": {
+            "interaction_selectpointinteraction_5896657031abe915479781": {
+                "serial": "interaction_selectpointinteraction_5896657031abe915479781",
+                "qtiClass": "selectPointInteraction",
+                "attributes": {
+                    "responseIdentifier": "RESPONSE",
+                    "maxChoices": 0,
+                    "minChoices": 0,
+                    "class": "responsive"
+                },
+                "debug": {
+                    "relatedItem": "item_589665700e6aa547084007"
+                },
+                "choices": {},
+                "prompt": {
+                    "serial": "container_containerstatic_5896657037d17805246232",
+                    "body": "\n            <p>Select the state of Iowa.<\/p>\n          ",
+                    "elements": {},
+                    "debug": {
+                        "relatedItem": "item_589665700e6aa547084007"
+                    }
+                },
+                "object": {
+                    "serial": "object_589665703bc0e529145041",
+                    "qtiClass": "object",
+                    "attributes": {
+                        "data": "assets\/us.jpg",
```

Comment:
For the unit test, could you please compress the image or use a lightweight one : 645KB is too much.